src/main/kotlin/org/rust/lang/core/resolve/NameResolution.kt

```diff
@@ -1025,13 +1025,22 @@ private fun processLexicalDeclarations(
         return processAll(boundNames, processor)
     }
 
-    fun processCondition(condition: RsCondition?, processor: RsResolveProcessor): Boolean {
-        if (condition == null || condition == cameFrom) return false
-        val pat = condition.pat
+    fun processPatList(patList: List<RsPat>, processor: RsResolveProcessor): Boolean {
```

Comment:
Due to the current name, someone might apply this function to any `patList` (e.g. `RsPatTup.patList`). Maybe rename it somehow to note that there is only one bind? Something like `processPatListBindOnce`.